styles/designs/webview/parts/_lists-components.scss

```diff
@@ -100,9 +100,7 @@ $StepwiseList__Container: (
 $StepwiseList__Item: (
     _name: "StepwiseListItem",
     _subselector: ' > li',
-    _properties: (
-      display: flex,
-    )
+    _properties: ()
 );
```

Comment:
When component doesn't have any property we use `empty_wrapper` function.
E.g. `$ChapterOutline: empty_wrapper(ChapterOutline, ' .os-chapter-outline');`